.classpath

```diff
@@ -1,14 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All of the changes in this file are reverting out mistakenly introduced classpath changes.